prep-example/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
+
+
```

Comment:
Hi Heather! Looks like your custom content was not included in the file. Be sure to let me know if I can help out!

Reply:
I am now stuck in the corrections.  It told me it was an invalid username and password and that authentication failed.
<img width="562" alt="Screen Shot 2021-09-20 at 6 34 35 PM" src="https://user-images.githubusercontent.com/90870500/134099005-75678038-af4f-42e2-8dc9-bcbea2132c34.png">

Reply:
<img width="538" alt="Screen Shot 2021-09-20 at 6 11 54 PM" src="https://user-images.githubusercontent.com/90870500/134110989-721dc319-3e12-4f29-8a33-02cea83b309e.png">